schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
-
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE SCHEMA `vet_clinic` ;
+CREATE TABLE `vet_clinic`.`animals` (
+  `idAnimals` INT NOT NULL AUTO_INCREMENT,
```

Comment:
- [ ] There is no keyword like `AUTO_INCREMENT` in PostgreSQL. Using auto_increment gives a syntax error when we try to create a table. If you want to auto-generate the id, you can do either of the following:

`idAnimals INT GENERATED ALWAYS AS IDENTITY,`
` idAnimals SERIAL PRIMARY KEY,`. Kindly fix this error. 👍